app/lib/shared/handlers.dart

```diff
@@ -59,6 +59,10 @@ shelf.Response htmlResponse(String content,
   headers ??= <String, String>{};
   headers['content-type'] = 'text/html; charset="utf-8"';
   headers['x-content-type-options'] = 'nosniff';
+  headers['content-security-policy'] = [
+    "frame-src 'none'",
+    "script-src 'self' https://www.googletagmanager.com/ https://www.google.com/ https://ajax.googleapis.com/",
```

Comment:
Maybe we should put these into a config file somewhere? Do we have a config file where the gtag is given?

Or at-least write them as a list...

Reply:
I've updated the source of the policy, and also added a few more items that were otherwise blocked on the staging. Unfortunately we still need the `unsafe-eval` policy, because `dart2js` still uses eval().